massclose.py

```diff
@@ -7,33 +7,72 @@
 workbook = xlrd.open_workbook('massclosurelist.xls')
 
 #Get input from user to find incident
-parent = input("Enter the parent incident ")
-
-#Search for incident
-#xpath = "C:\\Users\\hztzny\\AppData\\Local\\Programs\\Python\\Python36-32\\selenium\\webdriver\\firefox\\geckodriver.exe"
-xpath = "C:\\Users\\hztzny\\Desktop\\GSMC\\GeckoDriver\\chromedriver.exe"
-#Open up a Chrome browser and navigate to webpage
-driver = webdriver.Chrome(executable_path=xpath)
-#Open up ITSM website
-url = "https://itsm.gm.com"
-driver.get(url)
-#Click on Incident Managment tab
-incidentMan = driver.find_element_by_class_name("x-panel-header-text").click()

```

Comment:
You may run into issue with the class name because I had trouble with "x-panel-header-text" but mines could be for a different reason